src/domain/collection/__tests__/CollectionPageContainer.test.tsx

```diff
@@ -48,7 +51,11 @@ const getMocks = (
     variables: { draft, slug: collectionDetails.slug },
   }),
   getEventsByIdsMock({
-    variables: { ids: [curatedEventId], include: ['location'] },
+    variables: {
+      ids: [curatedEventId],
+      include: ['location'],
+      source: LinkedEventsSource.Linkedevents,
+    },
     eventsByIds,
   }),
 ];
```

Comment:
For completeness there could be a test with collection with both events and collections and we would check that both lists are visible?

Reply:
added course to "should show all events" test